admission/Makefile

```diff
@@ -42,6 +42,12 @@ endif
 	setup_envtest_env $(PWD)/$(ENVTEST_ASSETS_DIR) && \
 	TEST_PERMISSIVE=true go test -v -count 1 ./... ; \
 	}
+	{ \
+	source $(ENVTEST_ASSETS_DIR)/setup-envtest.sh && \
+	fetch_envtest_tools $(ENVTEST_ASSETS_DIR) && \
+	setup_envtest_env $(PWD)/$(ENVTEST_ASSETS_DIR) && \
+	NO_HTTPPROXY_MUTATION=true go test -v -count 1 ./... ; \
```

Comment:
`NO_HTTPPROXY_MUTATION=true` can be tested at the same time with `TEST_PERMISSIVE=true`.